wherescape/connectors/hubspot/process_data.py

```diff
@@ -15,52 +12,71 @@ def hubspot_process_results(
-    property_names = hubspot_instance.get_company_properties()
-    known_names = compare_names(column_names, property_names)
+    property_names = hubspot_instance.get_properties(object_name)
+    overlapping_names = compare_names(column_names, property_names)
 
+    """
+    Loops through results. Per row, the data is put into a dict after which this dict is added to the properties list.
+    Once the lists reaches 100 items (limit set by HubSpot's API), those get send as batch and the list is emptied for a new batch.
+    """
     for result in results:
-        """Hubspot only accepts 100 items per call"""
+        """hubspot only accepts 100 items per call"""
         if len(properties) < 100:
-            properties.append(create_data_dict(result, column_names, known_names))
+            properties.append(create_data_dict(result, column_names, overlapping_names))
         else:
             """
             send the collected data in patch, empty properties and start with the next results
             """
             logging.info("full batch ready")
-            send_data(object_name, request_type, properties, hubspot_instance)
+            send_data_to_hubspot(
+                object_name, request_type, properties, hubspot_instance
+            )
 
             properties.clear()
-            properties.append(create_data_dict(result, column_names, known_names))
+            properties.append(create_data_dict(result, column_names, overlapping_names))
 
     if len(properties) > 0:
         logging.info("final batch ready")
-        send_data(object_name, request_type, properties, hubspot_instance)
+        send_data_to_hubspot(object_name, request_type, properties, hubspot_instance)
 
 
-def send_data(
-    object_type: str, change_type: str, properties: list, hubspot_instance: Hubspot
+def send_data_to_hubspot(
+    object_type: str, request_type: str, properties: list, hubspot_instance: Hubspot
 ):
     """
-    Method to send data in the correct direction for
-    object_type (company, contact, deals) and
-    change_type (patch)
+    Function to send data in the correct direction
+
+    Parameters:
+    - object_type (string): refers to hubspot objects
+    - request_type (string): refers to the type of request being send
+    - properties (list): list of properties that will be updated
+    - hubspot_instance (Hubspot): HubSpot environment data will be sent to
     """
 
+    # hubspot_instance.send_patch(properties, object_type)
+
     if object_type == "companies":
-        if change_type == "patch":
-            hubspot_instance.send_company_patch(inputs=properties)
+        if request_type == "patch":
+            hubspot_instance.send_patch(properties, object_type)
     if object_type == "contacts":
-        if change_type == "patch":
-            hubspot_instance.send_contact_patch(inputs=properties)
+        if request_type == "patch":
+            hubspot_instance.send_contact_patch(properties, object_type)
```

Comment:
```suggestion
            hubspot_instance.send_patch(properties, object_type)
```
I think they can all use this method!